tsconfig.app.json

```diff
@@ -9,7 +9,7 @@
     "src/polyfills.ts"
   ],
   "include": [
-    "src/**/*.ts"
+    "src/**/*.d.ts"
```

Comment:
I haven't seen this before, but likely stretching the limits of my `tsc` expertise. Why include the typedef files?

Reply:
So `ng update` did this itself.  I'm going to assume Google knows more than I do, but I may dig into this more as I get a build/deploy going.

Reply:
SGTM. I'm down with "trust the tooling" :)